### PR TITLE
Fixes modal quote

### PR DESCRIPTION
**Pull request by steven-solar**
Previously the modal for changing a club's book read `Change the Club"s Book`, now it reads `Change the Club's Book`.

### CODE REVIEW

**Review by antoniolinhart — APPROVED** [review]
LGTM

File: capstone-frontend/src/components/ClubPage.jsx [hunk 1]
```diff
@@ -218,7 +218,7 @@ class ClubPage extends Component {
                       <SearchBookModal
                         objectId={this.props.id}
                         update={this.handleBookChange}
-                        text='Change the Club&quot;s Book'
+                        text='Change the Club&#39;s Book'
```
Comment (VivekShankar) [inline capstone-frontend/src/components/ClubPage.jsx:221]: Nice catch!

**Review by VivekShankar — APPROVED** [review]
LGTM

**Review by MauroGuerrero20 — APPROVED** [review]
LGTM

**Review by geoffbass — APPROVED** [review]
LGTM

File: capstone-frontend/src/components/ClubPage.jsx (hunk 1 above)
Comment (steven-solar) [reply capstone-frontend/src/components/ClubPage.jsx:221]: Thanks!

**Pull request merged**